posthog/warehouse/api/view_link.py

```diff
@@ -7,59 +7,54 @@
 from posthog.api.shared import UserBasicSerializer
 from posthog.hogql.database.database import create_hogql_database
 from posthog.models import User
-from posthog.warehouse.models import DataWarehouseSavedQuery, DataWarehouseViewLink
+from posthog.warehouse.models import DataWarehouseJoin
 
 
 class ViewLinkSerializer(serializers.ModelSerializer):
```

Comment:
Renaming here too

Reply:
Haven't got round to renaming _everything_ yet. It's currently a mismatch, but I'm keen to clean this up in a future PR. I did the database renaming now while we have no data though